src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -11,9 +11,11 @@
 import cryptography
 from cryptography import utils
 from cryptography.exceptions import InternalError
-from cryptography.hazmat.bindings._openssl import ffi, lib
+from cryptography.hazmat.bindings._rust import _openssl
 from cryptography.hazmat.bindings.openssl._conditional import CONDITIONAL_NAMES
 
+ffi = _openssl.ffi  # type: ignore
+lib = _openssl.lib  # type: ignore
```

Comment:
What actually uses these? Theoretically everything should go through a `Binding` instance.